System/Random.hs

```diff
@@ -956,16 +958,30 @@ instance UniformRange CDouble where
   uniformR (CDouble l, CDouble h) = fmap CDouble . uniformR (l, h)
 
 
+-- The `chr#` and `ord#` are the prim functions that will be called, regardless of which
+-- way you gonna do the `Char` conversion, so it is better to call them directly and
```

Comment:
I think this explanation is very useful. Thanks for adding it.

I wouldn't write "you gonna" as a code comment, I find it too casual. I'm happy to merge this as-is though and say we'll do a pass over comments to make the more consistent, something I think we should do anyway.

Reply:
Oh yeah, that's the American in me speaking. People worry less about things like that across the pond ;)
Fixed.